lib/donation_system/paypal_wrapper/creator_fields_generator.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'money'
+
+require_relative '../result'
+require_relative '../utils'
+
+module DonationSystem
+  module PaypalWrapper
+    class CreatorFieldsGenerator
+      INTENT = 'sale'
+      PAYPAL_METHOD = 'paypal'
+      NUMBER_PREFIX = 'P'
+
+      def self.execute(data)
+        new(data).execute
+      end
+
+      def initialize(data)
+        @data = data
+      end
+
+      def execute
+        {
+          intent: INTENT,
+          payer: { payment_method: PAYPAL_METHOD },
+          redirect_urls: redirect_urls,
+          transactions: [{
+            amount: { total: amount, currency: currency },
+            description: utils.generate_number(NUMBER_PREFIX)
+          }]
+        }
+      end
+
+      private
+
+      attr_reader :data
+
+      def amount
+        I18n.enforce_available_locales = false
```

Comment:
This will make this change globally? i.e. an unexpected side-effect of getting the amount.

Reply:
If you don't add that you get:
```
     I18n::InvalidLocale:
       :en is not a valid locale
```

Then there is this: https://github.com/RubyMoney/money/issues/593
and this: https://stackoverflow.com/questions/31416559/i18ninvalidlocale-en-is-not-a-valid-locale

If you know of a better way of fixing this let me know!

Reply:
I mean that this is *global* config for the I18n gem? In which case it should be set at application startup and not here.

Reply:
Also, `en:` should be a valid locale? Otherwise what is the I18n gem doing? ;) In which case the global config should be:

`I18n.config.available_locales = :en`